app/views/users/index.html.erb

```diff
@@ -1,3 +1,13 @@
-<h1>Here is a page for showing list of users</h1>
-
-<p>This page is under construction</p>
+<% @users.each do |user| %>
+  <div class="user_con">
+    <div class="user_img">
+      <%= image_tag("image5.jpg", width:'150px', height:'200px') %>
+    </div>
```

Comment:
- [ ] When viewing user page I am getting an error: 
`The asset "image5.png" is not present in the asset pipeline.`
Because in assets `image5` has jpg extension. Kindly address this error.:
<img width="1014" alt="Screenshot 2022-05-03 at 12 36 19" src="https://user-images.githubusercontent.com/77038610/166425261-5c026493-3b32-47f2-9912-43cad0dd7898.png">